backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -106,3 +106,94 @@ def test_add_review(self):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_review_requested_authorised_user(self):
+        actor_user = UserFactory(is_superuser=True, is_staff=True)
+        content_item = factories.ProjectContentItemFactory(
+            project_submission_type=ContentItem.LINK, template_repo=None
+        )
+        card = factories.AgileCardFactory(
+            status=AgileCard.READY,
+            recruit_project=factories.RecruitProjectFactory(content_item=content_item),
+            content_item=content_item,
+        )
+        self.login(actor_user)
+        card.start_project()
+        self.assertEqual(card.status, AgileCard.IN_PROGRESS)
+
+        request_review_url = f"{self.get_instance_url(card.id)}request_review/"
+        response = self.client.post(request_review_url)
+
+        self.assertEqual(response.status_code, 200)
+
+        card.refresh_from_db()
+        self.assertEqual(card.status, AgileCard.IN_REVIEW)
+
+        # sanity check
+        self.assertEqual(card.assignees.count(), 1)
+
+        self.assertEqual(LogEntry.objects.count(), 1)
+        entry = LogEntry.objects.first()
+
+        self.assertEqual(entry.actor_user, actor_user)
+        self.assertEqual(entry.effected_user, card.assignees.first())
+        self.assertEqual(entry.object_1, card.recruit_project)
+        self.assertEqual(entry.object_2, None)
+        self.assertEqual(entry.event_type.name, creators.CARD_REVIEW_REQUESTED)
+
+    def test_review_requested_assignee(self):
+
+        content_item = factories.ProjectContentItemFactory(
+            project_submission_type=ContentItem.LINK, template_repo=None
+        )
+        card = factories.AgileCardFactory(
+            status=AgileCard.READY,
+            recruit_project=factories.RecruitProjectFactory(content_item=content_item),
+            content_item=content_item,
+        )
+        self.login(card.assignees.first())
+        card.start_project()
+        self.assertEqual(card.status, AgileCard.IN_PROGRESS)
+
+        request_review_url = f"{self.get_instance_url(card.id)}request_review/"
+        response = self.client.post(request_review_url)
+
+        self.assertEqual(response.status_code, 200)
+
+        card.refresh_from_db()
+        self.assertEqual(card.status, AgileCard.IN_REVIEW)
+
+        # sanity check
+        self.assertEqual(card.assignees.count(), 1)
+
+        self.assertEqual(LogEntry.objects.count(), 1)
+        entry = LogEntry.objects.first()
+
+        self.assertEqual(entry.actor_user, card.assignees.first())
+        self.assertEqual(entry.effected_user, card.assignees.first())
+        self.assertEqual(entry.object_1, card.recruit_project)
+        self.assertEqual(entry.object_2, None)
+        self.assertEqual(entry.event_type.name, creators.CARD_REVIEW_REQUESTED)
+
+    def test_review_requested_non_authorised_user(self):
```

Comment:
Looking at this test my first comment was this;
> Not sure I get the point of this test, are we saying that only review requests from superusers will be logged? Looking at 
```
def log_project_competence_review_done(review):
``` 
> I don't think this is the case.

Then I realised you're testing that for someone who isn't an assignee nothing should be logged since they shouldn't be able to request reviews on other peoples stuff. Maybe this could be made more explicit, since we have `test_review_requested_assignee` right before, this makes it clear what the test is for: 
```suggestion
    def test_review_requested_by_non_assignee(self):
```

Also `is_superiser` is `False` by default for `UserFactory(is_superuser=False)` so these are equivalent
```
actor_user = UserFactory(is_superuser=False)
actor_user = UserFactory()
```
This `UserFactory()` is any regular user, not a super user and not staff.

Reply:
@TheAbstract thank you for the feedback :)